src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -127,10 +135,14 @@ const SubmissionDetails = _.flow(
    * Page render
    */
   return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
-    h(Link, {
-      href: Nav.getLink('workspace-job-history', { namespace, name }),
-      style: { alignSelf: 'flex-start', display: 'flex', alignItems: 'center', padding: '0.5rem 0' }
-    }, [icon('arrowLeft', { style: { marginRight: '0.5rem' } }), 'Back to list']),
+    div({ style: { marginBottom: '1rem', display: 'flex', alignItems: 'center' } }, [
+      h(Link, {
+        href: Nav.getLink('workspace-job-history', { namespace, name }),
+        style: { alignSelf: 'flex-start', display: 'inline-flex', alignItems: 'center', padding: '0.5rem 0' }
+      }, [icon('arrowLeft', { style: { marginRight: '0.5rem' } }), 'Job History']),
+      span({ style: { whiteSpace: 'pre' } }, [' > ']),
```

Comment:
* IIRC, because 
  * Logically this set of breadcrumbs is *within* the "Job History" tab.
  * There's lots more space to show it fully on the main page
  * It has plenty of space to grow if (eg) we add "why isn't my stuff running", or "call x / subworkflow y / call y / inputs" sub-pages
  * It works as a nice title for the page contents 
  * It replaces the weirdly out-of-place `Back to list` link.
* I'll try these two suggestions. I'm not great at comparing layouts for aesthetic comparisons but if it looks reasonably similar with your changes, I'll make them